config/jobs/kubernetes-sigs/structured-merge-diff/structured-merge-diff-presubmits.yaml

```diff
@@ -0,0 +1,18 @@
+presubmits:
+  kubernetes-sigs/structured-merge-diff:
+  - name: pull-structured-merge-diff-test
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
```

Comment:
For this job please add `decorate: true` above the `spec` field followed by: `path_alias: sigs.k8s.io/structured-merge-diff`
Then make the args something like:
```
command:
- go
args:
- test
- ./...
```

Reply:
Do I need to add `extra_refs` here as well?

Reply:
nope, presubmits know what repo and ref to checkout based on the commit they're testing in the PR, other jobs need to be told what you want, you _could_ add extra_refs here if you needed to check out another repo as well for some reason (some jobs do)